frappe/erpnext
# fix: Total taxes and charges in payment entry for multi-currency payments

Total Taxes and Charges for Party currency was not getting calculated properly

Before:
<img width="1337" alt="Screenshot 2022-02-28 at 7 04 29 PM" src="https://user-images.githubusercontent.com/42651287/155992273-75a3420e-5735-4755-90ed-4e7202ff6ff4.png">

After:
<img width="1326" alt="Screenshot 2022-02-28 at 7 04 51 PM" src="https://user-images.githubusercontent.com/42651287/155992292-d319f3ea-680c-41f0-ab64-b40670099426.png">

Steps to test:
1. Make a payment entry of type pay against a multicurrency supplier (not having paid to account currency same as company currency and paid from account having same as company)
2. Add some taxes and charges
3. Check if total taxes and charges show up correctly in both currencies (you can do the math)
